docs/tutorials/text_sentiment_ngrams.ipynb

```diff
@@ -0,0 +1,496 @@
+    "        Acc: 94.7%(train)       |       94.2%(valid)\n",
+    "        \n",
+    "Epoch: 6  | time in 0 minutes, 5 seconds\n",
+    "\n",
+    "        Loss: 0.0022(train)     |       0.0021(valid)\n",
+    "        Acc: 95.8%(train)       |       95.8%(valid)\n",
+    "        \n",
+    "Epoch: 7  | time in 0 minutes, 5 seconds\n",
+    "\n",
+    "        Loss: 0.0018(train)     |       0.0018(valid)\n",
+    "        Acc: 96.6%(train)       |       96.3%(valid)\n",
+    "        \n",
+    "Epoch: 8  | time in 0 minutes, 5 seconds\n",
+    "\n",
+    "        Loss: 0.0014(train)     |       0.0014(valid)\n",
+    "        Acc: 97.4%(train)       |       97.3%(valid)\n",
+    "        \n",
+    "Epoch: 9  | time in 0 minutes, 5 seconds\n",
+    "\n",
+    "        Loss: 0.0011(train)     |       0.0013(valid)\n",
+    "        Acc: 98.0%(train)       |       97.7%(valid)\n",
+    "        \n",
+    "Epoch: 10  | time in 0 minutes, 5 seconds\n",
+    "\n",
+    "        Loss: 0.0009(train)     |       0.0010(valid)\n",
+    "        Acc: 98.5%(train)       |       98.3%(valid)\n",
+    "        \n",
+    "Epoch: 11  | time in 0 minutes, 5 seconds\n",
+    "\n",
+    "        Loss: 0.0008(train)     |       0.0009(valid)\n",
+    "        Acc: 98.8%(train)       |       98.3%(valid)\n",
+    "        \n",
+    "Epoch: 12  | time in 0 minutes, 5 seconds\n",
+    "\n",
+    "        Loss: 0.0006(train)     |       0.0009(valid)\n",
+    "        Acc: 99.1%(train)       |       98.3%(valid)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluate the model with test dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('Checking the results of test dataset...')\n",
+    "with torch.no_grad():\n",
+    "    test_loss, test_acc = train_evaluate_func(model, test_dataset, loss_func,\n",
+    "                                              device=device, status='test')\n",
+    "print(f'\\tLoss: {test_loss:.4f}(test)\\t|\\tAcc: {test_acc * 100:.1f}%(test)')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Checking the results of test dataset...\n",
+    "\n",
+    "        Loss: 0.005(test)       |       Acc: 90.4%(test)\n",
+    "        \n",
+    "The results are consistent with the reference paper [Bag of Tricks for Efficient Text Classification](https://arxiv.org/abs/1607.01759)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test on a random news\n",
+    "\n",
+    "Use the best model so far and test a golf news."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import re\n",
+    "from torchtext.data.utils import ngrams_iterator\n",
+    "\n",
+    "ag_news_label = {1 : \"World\",\n",
```

Comment:
I'm wondering if we want to add "LABELS" field to torchtext.data.text_classification so that users can do torchtext.data.LABELS['AG_NEWS'][label]